helix-term/src/ui/prompt.rs

```diff
@@ -336,10 +335,7 @@ impl Prompt {
 
         let (range, item) = &self.completion[index];
 
-        // since we are using shellwords to parse arguments, make sure
-        // that whitespace in files is properly escaped.
-        let item = shellwords::escape(item);
-        self.line.replace_range(range.clone(), &item);
```

Comment:
Doesn't this undo the previous PR? The idea was that it would complete paths with spaces in quotes

Reply:
This part moved into the flatmap in `command_mode`: https://github.com/helix-editor/helix/pull/4587/files#diff-8f2b243886fc16c4a33efa5bd4ef54824301ed882ef62878ad9d4f6188d3d6a1R2227

The nice part of moving it there is that the completion results in the prompt show as escaped which matches how you have to input them (for example a file "a b.txt" would show as `a\ b.txt` in the prompt's completion items).